### DISCUSSION TITLE
cannot start server on ubuntu vm

### DISCUSSION THREAD

**Discussion by aj2620** [peers/peerjs#1221] [Q&A]
I'm trying to start an server on ubuntu but cannot I'm getting this error 

[image](https://user-images.githubusercontent.com/97669418/235356108-aadf2d1a-856f-4177-b690-df7bda7fd1c4.png)

when i did this on my local machine i had no errors everything was good but couldnt this to run on ubuntu vm 

thanks

**Comment by jonasgloning** [comment]
Maybe you are running an outdated version of Node on your Ubuntu VM?
What's the output of `node -v`
